cybergis_compute_client/UI.py

```diff
@@ -817,6 +910,11 @@ def rerender(self, components=[]):
             ct = ''.join(cl)
             getattr(self, 'render' + ct)()
 
+    """ Used to ensure that folders have names with only safe characters """
+    def makeNameSafe(self, text):
+        keepcharacters = (' ','.','_')
```

Comment:
Can somone abuse the naming scheme ? Are there any other characters that we might be missing ?